packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1554,6 +1554,7 @@ The following filters are supported with the filter query parameter:
 | `status eq "ACTIVE"`                | Apps that have a `status` of `ACTIVE`                                             |
 | `status eq "INACTIVE"`              | Apps that have a `status` of `INACTIVE`                                           |
 | `user.id eq ":uid"`                 | Apps assigned to a specific user such as `00ucw2RPGIUNTDQOYPOF`                   |
+| `name eq ":name"`                   | Instances of an app such as `template_swa`                                        |
```

Comment:
All other fields starts with "Apps ..." and here we suddenly switch to mentioning "Instances ...". For the purpose of this docs, an `App` object is always what we internally know as an instance. so we should stick with that. I.e. something like "Apps that have the same `name`, for example all `workday` apps"